README.md

```diff
@@ -184,38 +184,75 @@ This plugin creates device plugin endpoints based on the configurations given in
+| "resourcePrefix" | N        | Endpoint resource prefix name override.  Should not contain special characters                                    | `string` Default : "intel.com"                          |  "yourcompany.com"                                              |
+| "deviceType"     | N        | Device Type for a resource pool                                                                                   | `string` value of supported types. Default: "netDevice" | Currently supported values: "accelerator", "netDevice"          |
+| "selectors"      | N        | A map of device selectors                                                                                         | json object as `string` Default:  `null`                | Example: "selectors": {"vendors": ["8086"],"devices": ["154c"]} |
+
 
-|      Field       | Required |              Description               |                                       Type - Accepted values                                       |                                                Example/Accepted values                                                 |
-|------------------|----------|----------------------------------------|----------------------------------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------|
-| "resourceName"   | Y        | Endpoint resource name                 | string - should not contain special characters, must be unique in the scope of the resource prefix | "sriov_net_A"                                                                                                          |
-| "resourcePrefix" | N        | Endpoint resource prefix name override | string - should not contain special characters                                                     | "intel.com"                                                                                                            |
-| "selectors"      | N        | A map of device selectors              | Each selector is a map of string list.                                                             | "vendors": ["8086"],"devices": ["154c", "10ed"],"drivers": ["vfio-pci"],"pfNames": ["enp2s2f0"],"linkTypes": ["ether"] |
-| "isRdma"         | N        | Mount RDMA resources                   | `bool` - boolean value true or false                                                               | "isRdma": true                                                                                                         |
 
 Note: "resourceName" must be unique only in the scope of a given prefix, including the one specified globally in the CLI params, e.g. "example.com/10G", "acme.com/10G" and "acme.com/40G" are perfectly valid names.
 
-[//]: # (The table above generated using: https://ozh.github.io/ascii-tables/)
+#### Device selectors
```

Comment:
Can you also add a section describing what deviceType affects ?

Reply:
There is a description of deviceType in the table. What would you like to add?

Reply:
essentially what deviceType affects e.g  the types of selectors which are used.
there is some jump in logic imo straight to selectors in the README

Reply:
I've updated this section with additional explanation. PTAL

Reply:
Thanks!